engine/security/trust/trust_sandbox.md

```diff
@@ -61,150 +62,181 @@ and can be destroyed after you destroy the container.
+    ```
+
+3.  Create a file called `docker-compose.yml` with your favorite editor and add the following to the new file:
+
+    ```yaml
+    services:
+      notaryserver:
+        image: notary:server-0.7.0
+        networks:
+          - sandbox
+        depends_on:
+          - notarysigner
+        volumes:
+          - ./notary:/notarydir
+        command: |-
+          sh -c "/notarydir/migrations/migrate.sh &&
+          cd /notarydir/fixtures &&
+          notary-server -config=/notarydir/fixtures/server-config-local.json"
+
+      notarysigner:
+        image: notary:signer-0.7.0
+        networks:
+          - sandbox
+        volumes:
+          - ./notary:/notarydir
+        command: |-
+          sh -c "/notarydir/migrations/migrate.sh &&
+          cd /notarydir/fixtures &&
+          notary-signer -config=/notarydir/fixtures/signer-config-local.json"
+
+      trustsandbox:
+        image: docker:20.10.20-dind
+        networks:
+          - sandbox
         volumes:
-          notarycerts:
-            external: false
+          - ./notary:/notary
+        privileged: true
+        entrypoint: ""
+        command: |-
+            sh -c '
+                cp /notary/fixtures/root-ca.crt /usr/local/share/ca-certificates/root-ca.crt &&
+                update-ca-certificates &&
+                dockerd-entrypoint.sh --insecure-registry sandboxregistry:5000'
+
+      sandboxregistry:
+        image: registry:2.8.1
         networks:
-          sandbox:
-            external: false
+          - sandbox
 
-4. Save and close the file.
+    networks:
+      sandbox:
+        external: false
+    ```
+
+4.  Save and close the file.
 
-5. Run the containers on your local system.
+5.  Run the containers on your local system.
 
-        $ docker-compose up -d
+    ```console
+    $ docker compose up -d
+    ```
 
     The first time you run this, the docker-in-docker, Notary server, and registry
     images are downloaded from Docker Hub.
 
-
 ## Play in the sandbox
 
 Now that everything is setup, you can go into your `trustsandbox` container and
 start testing Docker content trust. From your host machine, obtain a shell
 in the `trustsandbox` container.
 
-    $ docker container exec -it trustsandbox sh
-    / #
+  ```console
+  $ docker compose exec trustsandbox sh
+  /#
+  ```
 
 ### Test some trust operations
 
 Now, pull some images from within the `trustsandbox` container.
 
-1. Download a `docker` image to test with.
-
-        / # docker pull docker/trusttest
-        docker pull docker/trusttest
-        Using default tag: latest
-        latest: Pulling from docker/trusttest
+1.  Download a `docker` image to test with.
 
-        b3dbab3810fc: Pull complete
-        a9539b34a6ab: Pull complete
-        Digest: sha256:d149ab53f8718e987c3a3024bb8aa0e2caadf6c0328f1d9d850b2a2a67f2819a
-        Status: Downloaded newer image for docker/trusttest:latest
+    ```console
+    /# docker pull docker/trusttest
```

Comment:
We should probably replace this with another image, because this image looks to be very old, and is still using the "v1" format (which is deprecated).

```bash
docker pull docker/trusttest
Using default tag: latest
latest: Pulling from docker/trusttest
Image docker.io/docker/trusttest:latest uses outdated schema1 manifest format. Please upgrade to a schema2 image for better future compatibility. More information at https://docs.docker.com/registry/spec/deprecated-schema-v1/
aac0c133338d: Pull complete
a3ed95caeb02: Pull complete
Digest: sha256:50c0cdd0577cc7ab7c78e73a0a89650b222f6ce2b87d10130ecff055981b702f
Status: Downloaded newer image for docker/trusttest:latest
docker.io/docker/trusttest:latest
```


For the examples on this page, "any" image should do, so we could just use `hello-world`, which is small, and part of the Docker official images.